php-frontend/src/main/java/org/sonar/plugins/php/api/tree/expression/MatchClauseTree.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * SonarQube PHP Plugin
+ * Copyright (C) 2010-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.php.api.tree.expression;
+
+import com.google.common.annotations.Beta;
+import org.sonar.php.api.PHPPunctuator;
+import org.sonar.plugins.php.api.tree.Tree;
+import org.sonar.plugins.php.api.tree.lexical.SyntaxToken;
+
+/**
+ * Starting from PHP 8.0
+ * <a href="https://wiki.php.net/rfc/match_expression_v2">Match expression</a>
+ */
+@Beta
+public interface MatchClauseTree extends Tree {
+
+  /**
+   * {@link PHPPunctuator#DOUBLEARROW =>}
+   */
+  SyntaxToken caseToExpressionToken();
```

Comment:
For consistency with `YieldExpressionTree`, `ForEachStatementTree`, and others, maybe we can call it `doubleArrowToken()`